src/ActionModal/components/ActionModal.vue

```diff
@@ -480,21 +479,6 @@ export default {
     prettyIncludedHeight() {
       return prettyInt(this.includedHeight)
     },
-    // TODO lets slice this monstrocity
-    context() {
-      return {
-        url: this.network.api_url,
-        networkId: this.network.id,
-        chainId: this.network.chain_id,
-        connected: this.connected,
-        userAddress: this.session.address,
-        rewards: this.rewards,
-        totalRewards: this.overview.totalRewards,
-        bondDenom: this.network.stakingDenom,
-        isExtensionAccount: this.isExtensionAccount,
-        account: this.overview.accountInformation
-      }
-    },
```

Comment:
I guess this deprecates your prior PR #3628 😅

Reply:
Directly into sendTxApi and simulateTxAPi